modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectValidationRuleConstants.java

```diff
@@ -16,6 +16,9 @@ public class ObjectValidationRuleConstants {
 
 	public static final String ENGINE_TYPE_GROOVY = "groovy";
 
+	public static final String ENGINE_TYPE_JAVA_DELEGATE_PREFIX =
```

Comment:
Why the engine name can't be just javaDelegate?
Why it's needed append the asset name to compose the key?